src/components/contact/assets/contact.scss

```diff
@@ -24,12 +24,13 @@ section.contact {
     width: 50%;
     margin-left: auto;
     text-align: right;
-    font-size: $font-xl;
-    color: $green300;
+    font-size: $font-xxl;
+    color: $green500;
+    font-weight: $font-weight-thin;
     text-transform: uppercase;
     border-bottom: 1px solid $gray;
     font-family: $font-fjalla-one;
-    margin-bottom: 0;
+    margin-bottom: $margin-none;
```

Comment:
```scss
margin-bottom: $margin-none;
```

`$margin-none` is still present here but has been removed from `_variables.scss`.